doc/api/synopsis.md

```diff
@@ -9,9 +9,62 @@ Please see the [Command Line Options][] document for information about
 different options and ways to run scripts with Node.js.
 
 ## Example
-
 An example of a [web server][] written with Node.js which responds with
-`'Hello World'`:
+`'Hello World!'`:
+
+Commands displayed in this document are shown starting with `$` or `>` 
+to replicate how they would appear in a user's terminal. 
+Do not include the `$` and `>` character.
+They are there to indicate the start of each command. 
+
+There are many tutorials and examples that follow this 
+convention: `$` or `>` for commands run as a regular user, and `#` 
+for commands that should be executed as an administrator. 
+
+Lines that don’t start with `$` or `>` character are typically showing 
+the output of the previous command.
+
+Firstly, make sure to have downloaded Node.js from [Node.js Official website][].
+
+To install Node using a package manager, see [this guide][].
+
+
+Now, create an empty project folder called `projects`, navigate into it:
+Project folder can be named base on user's current project title but
+this example will use `projects` as the project folder. 
+
+UNIX :
+
+```console
+$ mkdir ~/projects
+$ cd ~/projects
+```
+Windows CMD:
```

Comment:
It is always good to have a blank line before and after code / markdown blocks etc.